exporter/prometheusremotewriteexporter/config.go

```diff
@@ -52,6 +52,9 @@ type Config struct {
 
 	// SendMetadata controls whether prometheus metadata will be generated and sent
 	SendMetadata bool `mapstructure:"send_metadata"`
+
+	// RemoteWriteProtoMsg controls whether prometheus remote write v1 or v2 is sent
+	RemoteWriteProtoMsg RemoteWriteProtoMsg `mapstructure:"protobuf_message,omitempty"`
```

Comment:
Similar to the other comment, maybe we can simplify this to the user
```suggestion
	RemoteWriteVersion RemoteWriteVersion `mapstructure:"remote_write_version,omitempty"`
```

Reply:
Keeping the original as per https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35888#discussion_r1823250027